### ISSUE TITLE
Scaling window content

### ISSUE THREAD

**Issue by or** [chjj/compton#287]
Would compton be able to scale the content of a window? Basically making the window think it's inside a width x height rectangle, but rendering it at, say, (2 * width) x (2 * height) and scaling down any mouse events.

This would be useful for programs that can't deal with high density displays, in my case for instance certain wine applications.

I'm using i3 and asked about it there as well, but apparently the window manager is not capable of doing this, so I'm wondering whether the compositor could.

**Comment by ViktorNova** [comment]
This would be very cool

In the meantime, if you haven't tried it already, you may have some luck increasing the font DPI in winecfg, this has made a big difference for me a few times

**Comment by or** [comment]
I tried that, but unfortunately many applications don't honor the DPI settings.

**Comment by mingy0815** [comment]
@or: I wouldn't really mind that. As long as the windows are usable and in there correct size independently of the monitors native resolution I'd be ready to accept "blurry" windows. 

How about you guys?

**Comment by ViktorNova** [comment]
Absolutely, I think this would be fantastic! Pixely window contents would be no big deal, it's expected that when you zoom in 2x, something would be blurry

A related question: If this was implemented, would it be possible to have an option for window contents to scale when the window resizes as well, like Kwin does? So instead of updating window contents, stretch the current contents until the mouse/finger is released, then redraw the window at the new size
